rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -30,18 +30,32 @@
         {% endif %}
         <span class="plugin-author-link de-emphasized"><a target="_blank" href="https://openkat.nl" rel="noopener noreferrer">OpenKAT</a></span>
         {% if show_report_types %}
-            <div class="tags">
-                <p>
-                    <strong>{% translate "Required for:" %}</strong>
-                </p>
-                {% for plugin_id, report_types in plugin_report_types.items %}
-                    {% if plugin_id == plugin.id %}
-                        {% for report_type in report_types %}
-                            <span class="label tags-color-{{ report_type.label_style }}">{{ report_type.name }}</span>
-                        {% endfor %}
+            {% for plugin_id, report_types in plugin_report_types.items %}
+                {% if plugin_id == plugin.id %}
+                    {% if report_types|length >= 2 %}
+                        <div class="dropdown">
+                            <button class="dropdown-button ghost">
+                                {% translate "Required for" %} {{ report_types|length }} {% translate "report types" %}
+                                <span aria-hidden="true" class="icon ti-chevron-right"></span>
+                            </button>
+                            <div aria-expanded="false" class="dropdown-list">
```

Comment:
The `aria-expanded` here should be on the `button` rather than the `div`, and the `button` should have an `aria-controls` attribute that references the `div` by id. Ideally, the dropdown closes when the user presses Escape.

See for example the [Navigation Menu Button Example](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-links/) and [Example Disclosure Navigation Menu](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/) in the ARIA APG.

You could consider using Manon's `.collapsible` for this, see [Dropdown button](https://minvws.github.io/nl-rdo-manon/components/button-dropdown). This implements the `aria-expanded` behaviour, including open/close labels. It currently doesn't yet implement the Escape button behaviour.